### ISSUE TITLE
Clarify changes in Table 3

### ISSUE THREAD

**Issue by joseph-s-shapiro** [joseph-s-shapiro/erc#3]
Issues with Table 3 (ratio of marginal benefits of abatement to mean offset prices, 2010-2019):
- what happened to the tons and population weight X's at bottom rows of the table? Currently the table doesn't show how columns (1) and (2) are different, or how columns (3) and (4) are different. If we don't have space for those X's, we could always explain it in the table notes. But we should say it one way or another.
- Why did mean offset prices change so much between the version of this table we originally submitted (the one dated December 2020 on our website) and the current version? For example, the current T3 prices in the South are less than a third of those in the original version, but in the midwest both NOx and VOC prices are more than 10 times higher in the current versus the original version. I just want to make sure these changes are intended and have a good/known reason.

@reed-walker  (though I assigned this to @rayhkim which is github's way of asking Ray to take charge of this)

**Comment by rayhkim** [comment]
I added back the tons and population weight X's to T3 and AT3. T3 is a bit too long though, and goes into the margins of the paper.

As to why the mean offset prices changed so much, there are three reasons:
- Looking at an email chain from last year, it looks like there was a change made to the `combined_analysisSample.dta` file prior to my first using it which resulted in significant changes to the table, which I had noticed then as well. We discussed it over zoom and came to an agreement about the right way to create the `combined_analysisSample.dta` file. The one change that I do see between the August 2021 version of the `09_combine.do` file and the current one is that we now remove trades with price <= 10 before deflating prices, while it was done after deflating before—I'm not sure if there were further changes made between 2020 and then. This was the state of Table 2 after we had discussed the `combined_analysisSample.dta` last year:

<p align="center"><img width="500" alt="image" src="https://github.com/joseph-s-shapiro/erc/assets/6174670/c73fcbe3-432d-444f-9640-3bd26407e145"></p>

- In December, after discussing with Reed, I changed the code for columns 1 and 3 of Table 2 to "weight by tons within district (across years) and then again across district (using "district mean tons" as weights)" from the previous method of doing an amount-weighted mean over all trades. This made another substantial change to the columns 1 and 3 of the table:

<p align="center"><img width="500" alt="image" src="https://github.com/joseph-s-shapiro/erc/assets/6174670/22095c1d-5f37-4d4d-b7a3-5b077294562f"></p>

- I later removed trades occurring in districts without any nonattainment areas, which resulted in other minor changes to the table, giving us what we have now.
